includes/rest/class-users.php

```diff
@@ -80,6 +101,33 @@ public static function get( $request ) {
 		return $response;
 	}
 
+
+	/**
+	 * Endpoint for remote follow UI/Block
+	 *
+	 * @param WP_REST_Request $request The request object.
+	 *
+	 * @return void|string The URL to the remote follow page
+	 */
+	public static function remote_follow_get( WP_REST_Request $request ) {
+		$resource = $request->get_param( 'resource' );
+		$user_id  = $request->get_param( 'user_id' );
+
+		$template = Webfinger::get_remote_follow_endpoint( $resource );
+
+		if ( is_wp_error( $template ) ) {
+			return $template;
+		}
+
+		$resource = Webfinger::get_user_resource( $user_id );
```

Comment:
can we carry on with this resource? what about calls to a nonexistent/disabled `user_id` ?

Reply:
good point! missed to add the check :(